react/components/MiniCartContent.js

```diff
@@ -74,18 +76,24 @@ class MiniCartContent extends Component {
       0
     )
 
-  handleItemRemoval = async ({ id, cartIndex }) => {
-    const { updateItems } = this.props
+  handleItemRemoval = async ({ id, cartIndex }, index) => {
+    const { updateItems, updateLocalItems } = this.props
     const updatedItems = [
       {
         id,
-        index: cartIndex,
+        // the use of `!=` here is on purpose so we check for both
```

Comment:
unnecessary comment IMO

Reply:
I only added it because the first time I saw the `!=` I though it was a "typo" and I changed to use the `!==`, but that resulted in a bug :joy: 

but yeah, I think it's ok to remove it